tarantool/crud.py

```diff
@@ -0,0 +1,748 @@
+            self._crud_module_init()
+        assert self.require_ok == True, 'Problem with require of crud module on the remote machine'
+        assert isinstance(space_name, str)
+        assert isinstance(opts, dict)
+
+        crud_resp = self.con.call("crud.len", space_name, opts)
+
+        # NOTE: in absence of an error, crud does not give variable err as nil (as in most cases)
+        res, err = None, None
+        if len(crud_resp) == 1:
+            res = crud_resp[0]
+        else:
+            err = Error(crud_resp[1])
+
+        return res, err
+
+    def storage_info(self, opts={}):
+        """
+        TODO: write docs
+        """
+
+        if not self._crud_module_init_complete:
+            self._crud_module_init()
+        assert self.require_ok == True, 'Problem with require of crud module on the remote machine'
+        assert isinstance(opts, dict)
+
+        crud_resp = self.con.call("crud.storage_info", opts)
+
+        # NOTE: in absence of an error, crud does not give variable err as nil (as in most cases)
+        res, err = None, None
+        if len(crud_resp) == 1:
+            res = crud_resp[0]
+        else:
+            err = Error(crud_resp[1])
+
+        return res, err
+
+    def count(self, space_name, conditions={}, opts={}):
+        """
+        TODO: write docs
+        """
+
+        if not self._crud_module_init_complete:
+            self._crud_module_init()
+        assert self.require_ok == True, 'Problem with require of crud module on the remote machine'
+        assert isinstance(space_name, str)
+        assert isinstance(conditions, dict)
+        assert isinstance(opts, dict)
+
+        crud_resp = self.con.call("crud.count", space_name, conditions, opts)
+
+        res = None
+        if crud_resp[0] is not None:
+            res = crud_resp[0]
+
+        err = None
+        if crud_resp[1] is not None:
+            err = Error(crud_resp[1])
+
+        return res, err
+
+    def cfg(self, opts={}):
+        """
+        TODO: write docs
+        """
+
+        if not self._crud_module_init_complete:
+            self._crud_module_init()
+        assert self.require_ok == True, 'Problem with require of crud module on the remote machine'
+        assert isinstance(opts, dict)
+
+        crud_resp = self.con.call("crud.cfg", opts)
+
+        return Result(crud_resp.data[0])
+
+    def stats(self, space_name=None):
+        """
+        TODO: write docs
+        """
+
+        if not self._crud_module_init_complete:
+            self._crud_module_init()
+        assert self.require_ok == True, 'Problem with require of crud module on the remote machine'
+        if space_name is not None:
+            assert isinstance(space_name, str)
+
+        crud_resp = self.con.call("crud.stats", space_name)
+
+        res = None
+        if len(crud_resp.data[0]) > 0:
+            res = Result(crud_resp.data[0])
+
+        return res
+
+    def reset_stats(self):
```

Comment:
I think we also shouldn't provide `reset_stats` and `cfg` handles. Sure, it isn't hard for us to expose them, but I think that instance configuration is something that should be performed on server side. If one really would want to perform such things, they may always mess with call and eval.

On the other hand, having `stats` call shouldn't hurt anyone and we may leave it.

Reply:
Done, deleted `reset_stats` and `cfg`